aQute.libg/src/aQute/lib/io/IO.java

```diff
@@ -639,6 +636,35 @@ public static boolean isSymbolicLink(File link) {
 		return Files.isSymbolicLink(link.toPath());
 	}
 
+	/**
+	 * Creates a symbolic link from {@code link} to the {@code target}, or
+	 * copies {@code target} to {@code link} if running on Windows.
+	 * <p>
+	 * Creating symbolic links on Windows requires administrator permissions, so
+	 * copying is a safer fallback. Copy only happens if timestamp and and file
+	 * length are different than target
+	 *
+	 * @param link the location of the symbolic link, or destination of the
+	 *            copy.
+	 * @param target the target of the symbolic link, or source of the copy.
+	 * @return {@code true} if the operation succeeds, {@code false} otherwise.
+	 */
+	public static boolean createSymbolicLinkOrCopy(File link, File target) {
+		try {
+			if (isWindows() || !createSymbolicLink(link, target)) {
+				// only copy if target length and timestamp differ
+				if (target.lastModified() != link.lastModified() || target.length() != link.length()) {
+					IO.copy(target, link);
+					target.setLastModified(link.lastModified());
```

Comment:
Yes I did that backwards, partly because of confusing variable name.  I'll switch this up.